arches/app/views/api/__init__.py

```diff
@@ -1662,10 +1644,7 @@ def get(self, request):
 
             cardwidgets = [
                 widget
-                for widgets in [
-                    card.cardxnodexwidget_set.order_by("sortorder").all()
-                    for card in graph_cards
-                ]
+                for widgets in [card.cardxnodexwidget_set.all() for card in graph_cards]
```

Comment:
So removing this stuff is I think good because it reinforces that you don't have to do the custom `Prefetch` as highlighted in the previous comment.